### ISSUE TITLE
Cannot achieve good accuracy after training on COCO dataset

### ISSUE THREAD

**Issue by JohnPekl** [ChenhongyiYang/QueryDet-PyTorch#32]
I am trying to retrain your network. I set up a training environment following issue #25 . My computer has two GPU GeForce RTX 3090, 24G. I reduce batch size from your set up (16) to 8 in [train_mp.yaml](https://github.com/ChenhongyiYang/QueryDet-PyTorch/blob/main/models/querydet/configs/coco/train_mp.yaml) as follows.

```
SOLVER:
  # 3x
  # STEPS: (210000, 250000)
  # MAX_ITER: 270000

  # 1x
  BASE_LR: 0.005
  STEPS: (120000, 160000)
  MAX_ITER: 180000
  IMS_PER_BATCH: 8
```
After training, I load model_final.pth (change in [test.yaml](https://github.com/ChenhongyiYang/QueryDet-PyTorch/blob/main/models/querydet/configs/coco/test.yaml)). [The weight is uploaded here [GoogleDriver](https://drive.google.com/file/d/15LPaHmxa4YlmsuepuAa40HTsjfgSmAal/view?usp=sharing). Training log is [here](https://drive.google.com/file/d/1O94k6BWAGSpoXBDR-B4zHhvVDd0XP0OX/view?usp=sharing).]

```
MODEL:
  META_ARCHITECTURE: "RetinaNetQueryDet"
  WEIGHTS: "../default_dir/model_final.pth"
```

I then run [infer_coco.py.](https://github.com/ChenhongyiYang/QueryDet-PyTorch/blob/main/infer_coco.py), and use an image in COCO dataset (`datasets/coco/test2017/000000000001.jpg`). It gives me a weird result.

Do you know what went wrong?

![image](https://user-images.githubusercontent.com/15703029/176614984-f77c912c-3e54-45ab-99bc-54293ee78c20.png)

**Comment by ChenhongyiYang** [comment]
Hi, is this the baseline RetinaNet model? 

We used the default setting for RetinaNet provided by Detectron2 and the final AP should be around 37.3. We recommend you first try training the baseline and calibrating the numbers.

**Comment by 23jisuper** [comment]
> 我正在尝试重新训练您的网络。我在问题 #25 之后设置了一个培训环境。我的电脑有两个 GPU GeForce RTX 3090,24G。我在 [train_mp.yaml](https://github.com/ChenhongyiYang/QueryDet-PyTorch/blob/main/models/querydet/configs/coco/train_mp.yaml) 中将您的设置（16）的批处理大小减少到 8，如下所示。
> 
> ```
> SOLVER:
>   # 3x
>   # STEPS: (210000, 250000)
>   # MAX_ITER: 270000
> 
>   # 1x
>   BASE_LR: 0.005
>   STEPS: (120000, 160000)
>   MAX_ITER: 180000
>   IMS_PER_BATCH: 8
> ```
> 
> 训练后，我加载了 model_final.pth（[test.yaml](https://github.com/ChenhongyiYang/QueryDet-PyTorch/blob/main/models/querydet/configs/coco/test.yaml) 中的更改）。[权重上传至[此处 GoogleDriver](https://drive.google.com/file/d/15LPaHmxa4YlmsuepuAa40HTsjfgSmAal/view?usp=sharing)。训练日志[在这里](https://drive.google.com/file/d/1O94k6BWAGSpoXBDR-B4zHhvVDd0XP0OX/view?usp=sharing)。
> 
> ```
> MODEL:
>   META_ARCHITECTURE: "RetinaNetQueryDet"
>   WEIGHTS: "../default_dir/model_final.pth"
> ```
> 
> 然后我运行[infer_coco.py.](https://github.com/ChenhongyiYang/QueryDet-PyTorch/blob/main/infer_coco.py)，并使用COCO数据集（）中的图像。它给了我一个奇怪的结果。`datasets/coco/test2017/000000000001.jpg`
> 
> 你知道出了什么问题吗？
> 
> ![image](https://user-images.githubusercontent.com/15703029/176614984-f77c912c-3e54-45ab-99bc-54293ee78c20.png)

您好！  我在运行程序时始终说我数据集没有注册(coco2017)，然后看了看说可以使用内置数据集和注册自定数据集，整了两天还是加载不到数据集，能够指点指点吗  好绝望！！！
